app/views/queue_items/_queue_item.html.haml

```diff
@@ -1,18 +1,14 @@
-%tr
+%tr{ id: "queue_item #{queue_item.position}" }
+
   %td
-    %input.form-control(type="text" value="#{queue_item.position}")
+    = hidden_field_tag "queue[][id]", queue_item.id
+    = text_field_tag "queue[][position]", queue_item.position, class: 'form-control'
   %td
-    = link_to queue_item.video.title, queue_item.video
+    = link_to queue_item.video.title, queue_item.video, id: "#{queue_item.video.title}"
```

Comment:
Reaching through the video to get the title is a violation of the Law of Demeter. It would be better to setup a delegation or a method of your own:

```ruby
def video_title
  video.title
end
```

That reduces the coupling, allowing you to change that method without changing the view that uses it.